DotNet/QueryDispatch/Program.cs

```diff
@@ -165,17 +166,35 @@
 
 }
 
+var quartzProps = new NameValueCollection
+{
+    ["quartz.scheduler.instanceName"] = "DispatchScheduler",
+    ["quartz.scheduler.instanceId"] = "AUTO",
+    ["quartz.jobStore.clustered"] = "true",
+    ["quartz.jobStore.type"] = "Quartz.Impl.AdoJobStore.JobStoreTX, Quartz",
+    ["quartz.jobStore.driverDelegateType"] = "Quartz.Impl.AdoJobStore.SqlServerDelegate, Quartz",
+    ["quartz.jobStore.tablePrefix"] = "quartz.QRTZ_",
+    ["quartz.jobStore.dataSource"] = "default",
+    ["quartz.dataSource.default.connectionString"] = builder.Configuration.GetConnectionString(ConfigurationConstants.DatabaseConnections.DatabaseConnection),
+    ["quartz.dataSource.default.provider"] = "SqlServer",
```

Comment:
"SqlServer" is something configured in appSettings.json... Should this use that value, as well? I think we're fairly tied to SqlServer so I'm not sure it should be configurable. But, something to consider, in any event.